server/routes/routes.js

```diff
@@ -0,0 +1,100 @@
+const Product = require('Product model goes here');
+const User = require('User model goes here');
+const express = require('express');
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
+    try {
+        const user = await User.findOne({
+            where: {
+                sessionId: req.session.SID
+            }
+        })
+        if (user){
+            res.send(user)
+        }
+        next();
+    } catch (err){
+        console.error(err);
+    }
+})
+
+router.get('/products', async (req, res, next) => {
+    try {
+        res.send(await Product.findAll());
+    } catch (err){
+        console.error(err);
+    }
+});
+
+router.post('/products', async (req, res, next) => {
```

Comment:
if different HTTP requests are made to same route, consider using the `router.route('/products').get(...).post(...)` too reduce repetition